.github/workflows/pg-migtests.yml

```diff
@@ -163,6 +163,9 @@ jobs:
 
       - name: "TEST: pg-basic-non-public-live-migration-test"
         run: migtests/scripts/live-migration-run-test.sh pg/basic-non-public-live-test
+      
+      - name: "TEST: pg-unique-key-conflicts-test"
```

Comment:
Thanks for this test! Let's also run a fall-back OR fall-forward test? (for oracle as well). 
That will test the unique columns list for target YB.

Reply:
yeah planning to add test for fallf/fallb even for the conflicts detection also, in a next PR.

Reply:
In general, since we are aiming for smaller release cycles, I was thinking of moving away from this practice of deferring tests to later PRs. I think it's okay if it takes longer(at worst, the feature will come in the next release), but for new features at least, we should ensure that they are all automation-tested. 
That is the only way we we will be able to sustain 2-week releases.
Let's discuss this in stand-up!

Reply:
for this specific case, fallf and fallforward, @shubham-yb is taking up, since i don't have enough bandwidth.
But adding fallf and fallb is on the TODO list before this bug fix goes out in release.